pynicotine/gtkgui/userbrowse.py

```diff
@@ -191,10 +197,11 @@ def __init__(self, userbrowses, user):
         # Iters for current FileStore
         self.files = {}
         self.totalsize = 0
+        self.num_selected_files = 0
 
         self.dir_store = Gtk.TreeStore(str)
         self.FolderTreeView.set_model(self.dir_store)
-        self.folder_key_controller = connect_key_press_event(self.FolderTreeView, self.on_folder_key_press_event)
+        self.folder_key_controller = connect_key_press_event(self.FolderTreeView, self.on_key_press_event_folder)
```

Comment:
This follows the naming scheme in other modules, if you need me to change it back let me know.